infrastructure/simulation-tool/src/simulations/withdraw.ts

```diff
@@ -0,0 +1,53 @@
+import {
+    PreparedWithdrawal,
+    executeWithdrawals,
+    generateWithdrawals,
+    resolveWithdrawTransactions
+} from '../operations/withdraw';
+import { SimulationConfiguration } from './setup';
+import config from '../utils/config.utils';
+import { Wallet as RollupWallet, Transaction } from '@rsksmart/rif-rollup-js-sdk';
+import { generateWallets } from '../utils/wallet.utils';
+import { ensureL2AccountActivation, ensureRollupFunds, ensureRollupFundsFromRollup } from '../operations/common';
+import { BigNumber, ethers } from 'ethers';
+
+const runSimulation = async ({ txCount, txDelay, walletGenerator, funderL2Wallet }: SimulationConfiguration) => {
+    const gasCost = ethers.utils.parseEther('0.001');
```

Comment:
when ran in production this could be unnecessarily high, perhaps? In another PR I get the gas factor for each account separately in the funding stage. I don't actually know whether that is better, tho...